src/themes.ts

```diff
@@ -0,0 +1,16 @@
+const theme = {
+
+    colors:{
+        main: '#385775',
+        title: '#fff',
```

Comment:
[Suggestion] You should use the color name if it exist:
![image](https://user-images.githubusercontent.com/20242580/214043428-b92940a0-1225-4b41-bbbb-d0df3ab595ea.png)